crates/bevy_derive/src/lib.rs

```diff
@@ -1,4 +1,5 @@
-#![expect(missing_docs, reason = "Not all docs are written yet, see #3492.")]
+//! Assorted proc macro derive functions.
```

Comment:
This feels a bit lacklustre, but I couldn't think of anything better - that appears to be what this crate _is_ for!

Reply:
Yeah lol, that's about the best I've got too!